net/udp/udp_recvfrom.c

```diff
@@ -351,11 +352,13 @@ static inline void udp_sender(FAR struct net_driver_s *dev,
           else
 #endif
             {
+              FAR socklen_t *fromlen = pstate->ir_fromlen;
```

Comment:
maybe we can have this defined at line 324. I mean define it once and clean-up `if (conn->domain == PF_INET6)` and `else` case?